File: 프로그래머스/0/181935. 홀짝에 따라 다른 값 반환하기/홀짝에 따라 다른 값 반환하기.py
def solution(n):
    answer = 0
    
    if n % 2 == 0 :
        for i in range(2, n+1, 2):
            answer += i * i
    else:
        for j in range(1, n+1, 2):
            answer += j;
    
    return answer

File: 프로그래머스/0/181935. 홀짝에 따라 다른 값 반환하기/홀짝에 따라 다른 값 반환하기.c
#include <stdio.h>
#include <stdbool.h>
#include <stdlib.h>

int solution(int n) {
    int answer = 0;
    
    if(n % 2 == 0){
        for(int i=1;i<=n;i++){
            if(i % 2 == 0){
                answer += i * i;
            }
        }
    }
    
    else{
        for(int j=1;j<=n;j++){
            if(j % 2 != 0){
                answer += j;
            }
        }
    }
    
    return answer;
}